data.yaml

```diff
@@ -1,4 +1,22 @@
 alerts:
+  - identifier:
+    channel: operator
+    approved_at: 2021-09-09T10:15:00+01:00
+    starts_at: 2021-09-09T10:15:00+01:00
+    cancelled_at: 2021-09-09T10:30:00+01:00
+    finishes_at: 2021-09-09T10:30:00+01:00
+    content: |
+      This is a mobile network operator test of the Emergency Alerts service. You do not need to take any action. To find out more, search for gov.uk/alerts
+    areas: {}
```

Comment:
is it intended that areas are empty?

Reply:
I copied all the other operator alerts below which are the same so I assume it is OK because we don't show which areas operator tests get sent to.

If it was on a severe channel then it would matter.

Reply:
Makes sense, thanks! 👍🏼